recipes/pcapplusplus/all/patches/0004-Fix-md5-include.patch

```diff
@@ -0,0 +1,11 @@
+--- mk/vs/Packet++.vcxproj.template
++++ mk/vs/Packet++.vcxproj.template
+@@ -271,7 +271,7 @@
+   </ItemGroup>
+   <ItemGroup>
+     <ClCompile Include="..\..\3rdParty\hash-library\md5.cpp" />
+-    <ClCompile Include="..\..\3rdParty\hash-library\md5.h" />
++    <ClInclude Include="..\..\3rdParty\hash-library\md5.h" />
```

Comment:
If this project is not in CCI can you please add a requires with e `#TODO` so we don't forget about it please 🙏